packages/node_modules/@webex/internal-plugin-ediscovery/src/ediscovery.js

```diff
@@ -193,51 +193,38 @@ const EDiscovery = WebexPlugin.extend({
     // use spread operator to set default options
     options = {...defaultOptions, ...options};
 
-    const promises = [];
-
-    // The content summary for a report is needed in order to decrypt the content for that report
-    // Since retrieving it from the service every time it is needed is extremely inefficient,
-    // we ensure that is is cached here before proceeding to get the report contents
-    if (!this.contentSummaryCache.has(reportId)) {
-      promises.push(this.getContentSummary(reportId));
-    }
-
-    return Promise.all(promises)
-      .then(() => this.request({
-        method: 'GET',
-        service: 'ediscovery',
-        resource: `reports/${reportId}/contents`,
-        qs: {offset: options.offset, size: options.size},
-        timeout: options.timeoutMs
-      }));
+    return this.request({
+      method: 'GET',
+      service: 'ediscovery',
+      resource: `reports/${reportId}/contents`,
+      qs: {offset: options.offset, size: options.size},
+      timeout: options.timeoutMs
+    });
   },
 
   @waitForValue('@')
-  @oneFlight({keyFactory: ({reportId}) => `${reportId}`})
+  @oneFlight({keyFactory: (reportId) => `${reportId}`})
```

Comment:
This error may explain some of the unusual behaviour we've seen around getContentSummary

Reply:
where the error was the extra set of braces in the annotation